modules/base/index.ts

```diff
@@ -1,6 +1,6 @@
 export { default as extendStyles } from "./extendStyles";
 export { default as useStyles } from "./useStyles";
 export { default as useTheme } from "./useTheme";
-export { default as withStyles } from "./withStyles";
+export { default as withStyles, withStylesHOC } from "./withStyles";
```

Comment:
do we use withStylesHOC for anything?

Reply:
on ddm, this is only because ts types. which isn't perfect but was the solution I found :/